packages/components/organisms/f-checkout/src/services/tests/mapper.test.js

```diff
@@ -60,9 +60,30 @@ describe('checkout mapper', () => {
         expect(locationRequest.address.locality).toBe(address.locality);
         expect(locationRequest.address.lines).toStrictEqual([
             address.line1,
-            address.line2,
-            null,
-            null
+            address.line2
+        ]);
+    });
+
+    it('should not map the second line of the address when it\'s not provided', () => {
```

Comment:
Pro tip – if you use `’` (which is `Option + Shift + ]` on Mac – no idea on Windows, but I'm sure there is a way…), you don't need to escape these sort of apostrophes and you get a proper styled grammatical apostrophe too 🚀